Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -99,9 +99,10 @@ you may perform the following steps to troubleshoot ClusterMesh issues.
       containers in the clustermesh-apiserver deployment, to authenticate against the
       sidecar etcd instance. Not applicable if an external etcd cluster is used.
 
-    * ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents, and
-      optionally the kvstoremesh container in the clustermesh-apiserver deployment,
-      to authenticate against remote etcd instances (either internal or external).
+    * ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents (or
+      the kvstoremesh container in the clustermesh-apiserver deployment, when
+      KVStoreMesh is enabled) to authenticate against remote etcd instances
+      (either internal or external).
```

Comment:
Can we remove the parenthesis around the comment about the kvstoremesh container? Also, small nit regarding the wording in the last sentence:

```suggestion
    * ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents, or
      the kvstoremesh container in the clustermesh-apiserver deployment when
      KVStoreMesh is enabled, to authenticate against ClusterMesh etcd instances
      in both local and remote clusters.
```

Reply:
ACK for the parenthesis. Differently, while I agree that the `"either internal or external"` wording may be confusing, I don't think that the reworded sentence preserves the same meaning. AFAIU, internal/external in this context refers to whether Cilium is configured in CRD (hence with the clustermesh-apiserver -- internal) or kvstore mode (external) and I'd propose to just drop it:

```
     ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents, or
      the kvstoremesh container in the clustermesh-apiserver deployment, when
      KVStoreMesh is enabled, to authenticate against remote etcd instances.
```

Reply:
Ah ok I interpreted it differently. Your rewording looks good, but can we remove the comma between `deployment` and `when`?

```
     ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents, or
      the kvstoremesh container in the clustermesh-apiserver deployment when
      KVStoreMesh is enabled, to authenticate against remote etcd instances.
```